src/main/scripts/org/reaktivity/specification/mqtt/connection/client.rpt

```diff
@@ -25,9 +25,9 @@ write [0x10 0x11]           # fixed header
       [0x00 0x04] "MQTT"    # protocol name
       [0x05]                # protocol version
       [0x02]                # connect flags = clean start
-      [0x00 0x3C]           # keep alive = 60s
+      [0x00 0x3c]           # keep alive = 60s
       [0x00]                # properties = none
-      [0x00 0x04] "abcd"    # clientID
+      [0x00 0x04] "abcd"    # clientId
```

Comment:
Let's use something like `"client"` as the `clientId` for the test scripts instead of `"abcd"`.